bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v18/BitcoindV18RpcClientTest.scala

```diff
@@ -0,0 +1,86 @@
+package org.bitcoins.rpc.v18
+import org.bitcoins.chain.models.BlockHeaderDbHelper
+import org.bitcoins.core.protocol.blockchain.RegTestNetChainParams
+import org.bitcoins.rpc.client.common.BitcoindVersion
+import org.bitcoins.rpc.client.v18.BitcoindV18RpcClient
+import org.bitcoins.testkit.chain.BlockHeaderHelper
+import org.bitcoins.testkit.rpc.BitcoindRpcTestUtil
+import org.bitcoins.testkit.util.BitcoindRpcTest
+
+import scala.concurrent.Future
+
+class BitcoindV18RpcClientTest extends BitcoindRpcTest {
+  lazy val clientF: Future[BitcoindV18RpcClient] = {
+    val client = new BitcoindV18RpcClient(BitcoindRpcTestUtil.v18Instance())
+    val clientIsStartedF = BitcoindRpcTestUtil.startServers(Vector(client))
+    clientIsStartedF.map(_ => client)
+  }
+
+  clientF.map(c => clientAccum.+=(c))
+
+  behavior of "BitcoindV18RpcClient"
+
+  it should "be able to start a V18 bitcoind instance" in {
+
+    clientF.map { client =>
+      assert(client.version == BitcoindVersion.V18)
+    }
+
+  }
+
+  it should "return active rpc commands" in {
+    clientF.map(client =>
+      client.getNewAddress.flatMap(client.generateToAddress(50000, _)))
```

Comment:
should be
```
val generatedF = clientF.flatMap(client => client.getNewAddress.flatMap(client.generateToAddress(50000, _)))

val rpcinfoF = generatedF.flatMap(...
```
otherwise the test will not wait for the generate call